nucypher_cli/.nucypher.ini

```diff
@@ -0,0 +1,49 @@
+; ==================================
+;     ~ MACHINE TEMPLATE ONLY ~
+;
+;   DO NOT USE THIS CONFURATION FILE.
```

Comment:
`CONFURATION` -> `CONFIGURATION`

Reply:
DON'T YALL GO 'ROUND USIN' A THIS A' HERE CONFURATION FILE.

Reply:
setting are confu